backend/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "dsek-backend",
+  "main": "gateway.js",
+  "scripts": {
+    "service:news": "nodemon --watch ./services/news ./services/news/index.js",
+    "service:members": "nodemon --watch ./services/members ./services/members/index.js",
+    "services": "concurrently -k npm:service:*",
```

Comment:
kan vara najs att lägga in ett alias här så npm start också funkar, bara för muskelminnet från de andra

Reply:
Yes, väldigt rimligt